email/src/main/resources/chart.email/index.scala.html

```diff
@@ -0,0 +1,3 @@
+@(atom: com.gu.contentatom.thrift.Atom
+, data: com.gu.contentatom.thrift.atom.chart.ChartAtom
+)
```

Comment:
Perhaps one day 🤞 

Can we move this into `/chart/email` instead of `/chart.email`?

Reply:
Oops! I hadn't noticed the directory structure 😄 Fixed now.